apps/console/src/features/claims/components/wizard/local-claim/mapped-attributes.tsx

```diff
@@ -125,24 +125,29 @@ export const MappedAttributes: FunctionComponent<MappedAttributesPropsInterface>
-                                            { store.name }
-                                        </Grid.Column>
-                                        <Grid.Column width={ 12 }>
-                                            <Field
-                                                type="text"
-                                                name={ store.name }
-                                                placeholder={ t("console:manage.features.claims.local.forms." +
-                                                    "attribute.placeholder") }
-                                                required={ true }
-                                                requiredErrorMessage={ t("console:manage.features.claims.local.forms." +
-                                                    "attribute.requiredErrorMessage") }
-                                                value={ values?.get(store.name)?.toString() }
-                                                data-testid={ `${ testId }-form-store-name-input` }
-                                            />
-                                        </Grid.Column>
-                                    </Grid.Row>
+                                    <>
+                                        { store?.enabled && (
+                                            <Grid.Row columns={ 2 } key={ index }>
+                                                <Grid.Column className="centered-text" width={ 4 }>
+                                                    { store.name }
+                                                </Grid.Column>
+                                                <Grid.Column width={ 12 }>
+                                                    <Field
+                                                        type="text"
+                                                        name={ store.name }
+                                                        placeholder={ t("console:manage.features.claims.local.forms." +
+                                                            "attribute.placeholder") }
+                                                        required={ true }
+                                                        requiredErrorMessage={ t("console:manage.features.claims." +
+                                                            "local.forms.attribute.requiredErrorMessage") }
+                                                        value={ values?.get(store.name)?.toString() }
+                                                        data-testid={ `${ testId }-form-store-name-input` }
+                                                    />
+                                                </Grid.Column>
+                                            </Grid.Row>
+                                        ) }
+                                    </>
+                                    
```

Comment:
remove this line

Reply:
Fixed by f6a0cee2ab55da211fbc3fcf3c6e4dfc694f3c9c.